packages/ui/src/lib/Action/Action.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { forwardRef } from 'react';
+import { Button, ButtonProps, LinkButton, LinkButtonProps } from '../Button';
+
+export type ActionProps = ButtonProps &
+  LinkButtonProps &
+  (
+    | { /** The path or link to be used. Use this if 'to' is not used */ href: string; to?: never }
+    | { /** The rout to navigate to using react-router. Use this if 'href' is not used */ to: string; href?: never }
+    | { /** Either 'to' or 'href' should be used, not both */ to?: never; href?: never }
+  );
+
+/**
+ * @title
+ *  Action Button
+ * @label
+ *  Action Button
+ * @shortdescription
+ *  Clover styled Action Button.
+ * @keywords
+ *  [Button,Action,Action Button, html button, anchor]
+ * @description
+ * Clover styled Action Button. Note: Use sentence case for the label!
+ * @componentUsage
+ * <Action variant="secondary">
+    Secondary click
+   </Action>
+ * @example
+    <ActionButtonTemplate />
+ */
+export const Action = forwardRef<HTMLButtonElement | HTMLAnchorElement, ActionProps>(
+  ({ href, to, ...passThroughProps }, ref) => {
+    return (
+      <>
```

Comment:
no need for a fragment here

Reply:
i see what you mean, but i will need to refactor da f out of these, but for now scope is limited to just being able to export the Action component independently.